docs/ref/models/querysets.txt

```diff
@@ -2572,23 +2572,25 @@ in the database <database-time-zone-definitions>`.
 hour
 ~~~~
 
-For datetime fields, an exact hour match. Allows chaining additional field
```

Comment:
Should add `.. versionchanged:: 1.9` annotations to describe the new functionality and also a mention in the 1.9 release notes.